solana-programs/anchor/audius-data/rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import resolve from '@rollup/plugin-node-resolve'
+import typescript from '@rollup/plugin-typescript'
+
+import pkg from './package.json'
+
+const extensions = ['.js', '.ts']
+
+export default {
+  input: 'lib/lib.ts',
+  output: [{ file: pkg.main, format: 'cjs' }],
+  plugins: [
+    commonjs({ extensions }),
+    json(),
+    resolve({ extensions, preferBuiltins: true }),
+    typescript({include: ["./lib/lib.ts", "./lib/utils.ts"]})
```

Comment:
would we also want target/types/audius_data.ts?

Reply:
and should this build relate to the client / IDL from anchor build?

Reply:
yeah so im thinking for first pass we just rollup things to js and not publish types. in this case that means we don't need to actually include audius_data.ts, since that's just type info. what do you think?

Reply:
ah ok sounds good